client/app/startup/SearchPageApp.js

```diff
@@ -56,7 +73,15 @@ export default (props) => {
     currentPage: rawListings.map((l) => l.get(':id')),
   });
 
-  const combinedProps = Object.assign({}, { marketplace: props.marketplace }, { searchPage, routes, listings, profiles });
+  const flashNotifications = systemNotificationsToList(props.marketplace.notifications);
```

Comment:
Now we have the notifications in two different parts of redux state: `flashNotifications` and `marketplace.notifications`. Not sure how this should be fixed, but it's not very clean now.